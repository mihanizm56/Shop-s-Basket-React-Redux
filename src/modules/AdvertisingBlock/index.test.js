import React from "react"
import { shallow } from "enzyme"
import { AdvertisingBlock } from "./index"

describe("Youtube", () => {
  const testArray = [
    {
      name: "test_name_1",
      price: "1234",
    },
    {
      name: "test_name_2",
      price: "1234",
    },
    {
      name: "test_name_3",
      price: "1234",
    },
  ]

  it("Renders correct defaultProps", () => {
    const component = shallow(<AdvertisingBlock />)

    expect(component).toMatchSnapshot()
  })

  it("Renders correct defaultProps", () => {
    const component = shallow(<AdvertisingBlock addList={testArray} />)

    expect(component).toMatchSnapshot()
  })
})
