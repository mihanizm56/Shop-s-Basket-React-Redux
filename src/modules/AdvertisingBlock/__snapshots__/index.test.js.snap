// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Youtube Renders correct defaultProps 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <styled.h2>
        Добавьте к вашему заказу
      </styled.h2>
    </styled.div>
    <styled.div>
      <styled.div
        key="0"
      >
        <styled.div
          style={
            Object {
              "backgroundImage": "url(good-1.png)",
            }
          }
        />
        <styled.div>
          test_name
        </styled.div>
        <styled.div>
          test_price
           руб.
        </styled.div>
      </styled.div>
    </styled.div>
  </styled.div>
</styled.div>
`;

exports[`Youtube Renders correct defaultProps 2`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <styled.h2>
        Добавьте к вашему заказу
      </styled.h2>
    </styled.div>
    <styled.div>
      <styled.div
        key="0"
      >
        <styled.div
          style={
            Object {
              "backgroundImage": "url(good-1.png)",
            }
          }
        />
        <styled.div>
          test_name_1
        </styled.div>
        <styled.div>
          1234
           руб.
        </styled.div>
      </styled.div>
      <styled.div
        key="1"
      >
        <styled.div
          style={
            Object {
              "backgroundImage": "url(good-2.png)",
            }
          }
        />
        <styled.div>
          test_name_2
        </styled.div>
        <styled.div>
          1234
           руб.
        </styled.div>
      </styled.div>
      <styled.div
        key="2"
      >
        <styled.div
          style={
            Object {
              "backgroundImage": "url(good-3.png)",
            }
          }
        />
        <styled.div>
          test_name_3
        </styled.div>
        <styled.div>
          1234
           руб.
        </styled.div>
      </styled.div>
    </styled.div>
  </styled.div>
</styled.div>
`;
